Lib9c.Tools/Utils.cs

```diff
@@ -0,0 +1,64 @@
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Libplanet;
+using Libplanet.Action;
+using Libplanet.Blocks;
+using Libplanet.Crypto;
+using Nekoyume.Action;
+using Nekoyume.Model;
+using Nekoyume.Model.State;
+
+namespace Lib9c.Tools
+{
+    public static class Utils
+    {
+        public static void ExportBlock(Block<PolymorphicAction<ActionBase>> block, string path)
+        {
+            byte[] encoded = block.Serialize();
+            File.WriteAllBytes(path, encoded);
+        }
+
+        public static void ExportKeys(List<ActivationKey> keys, string path)
+        {
+            File.WriteAllLines(path, keys.Select(v => v.Encode()));
+        }
+
+        public static Dictionary<string, string> ImportSheets(string dir)
+        {
+            var sheets = new Dictionary<string, string>();
+            var files = Directory.GetFiles(dir, "*.csv", SearchOption.AllDirectories);
+            foreach (var filePath in files)
+            {
+                var fileName = Path.GetFileName(filePath);
+                if (fileName.EndsWith(".csv"))
+                {
+                    fileName = fileName.Split(".csv")[0];
```

Comment:
I wonder why to split it with *.csv*. If the purpose was to remove the extension, *.csv*, then how about using [`String.Substring`][string-substring] or [`String.Remove`][string-remove]? It is not a comment for performance, there seems [some benchmark posted in StackOverflow by user][some-benchmark].

[some-benchmark]: https://stackoverflow.com/a/54896713
[string-remove]: https://docs.microsoft.com/en-us/dotnet/api/system.string.remove?view=netcore-3.1
[string-substring]: https://docs.microsoft.com/en-us/dotnet/api/system.string.substring?view=netcore-3.1

Reply:
How about using `Path.GetFileNameWithoutExtension(filePath);`?

Reply:
I'll use `Path.GetFileNameWithoutExtension(filePath);`

Reply:
amended by 9ac8f6e